src/crimson/os/seastore/transaction_manager.h

```diff
@@ -643,6 +602,86 @@ class TransactionManager : public ExtentCallbackInterface {
     ExtentPlacementManager::dispatch_result_t dispatch_result,
     std::optional<journal_seq_t> seq_to_trim = std::nullopt);
 
+  /**
+   * pin_to_extent
+   *
+   * Get extent mapped at pin.
+   */
+  using pin_to_extent_iertr = base_iertr;
```

Comment:
Hi, why move this part to 'private'? I doubt this because I got 'make' error, but it works fine if I revert this part to 'public'.

Reply:
error: ‘crimson::os::seastore::TransactionManager::pin_to_extent_ret<T> crimson::os::seastore::TransactionManager::pin_to_extent(crimson::os::seastore::Transaction&, crimson::os::seastore::LBAMappingRef) [with T = crimson::os::seastore::TestBlock; crimson::os::seastore::TransactionManager::pin_to_extent_ret<T> = crimson::interruptible::interruptible_future_detail<crimson::os::seastore::TransactionConflictCondition, crimson::errorator<crimson::unthrowable_wrapper<const std::error_code&, ((const std::error_code&)(& crimson::ec<5>))> >::_future<crimson::errorated_future_marker<boost::intrusive_ptr<crimson::os::seastore::TestBlock> > > >; crimson::os::seastore::LBAMappingRef = std::unique_ptr<crimson::os::seastore::PhysicalNodeMapping<long unsigned int, crimson::os::seastore::paddr_t>, std::default_delete<crimson::os::seastore::PhysicalNodeMapping<long unsigned int, crimson::os::seastore::paddr_t> > >]’ is private within this context

Reply:
@Svelar just fixed it:-) it seems that the default built items doesn't include store_nbd, so I didn't see this error.

Reply:
Cool!